mappings/net/minecraft/world/gen/decorator/Decorator.mapping

```diff
@@ -0,0 +1,50 @@
+CLASS byb net/minecraft/world/gen/decorator/Decorator
+	FIELD A HELL_FIRE Lbyb;
```

Comment:
Do mojang/other pomf places still call it hell or is called neather in other places?

Reply:
I named everything based on the string ID Mojang gave everything. Some of the names are a bit wonky... not sure why

Reply:
I guess if thats what mojang call then its fine. Might still be worth checking what other places in POMF use